app/svelte/package.json

```diff
@@ -32,11 +33,13 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
+    "@storybook/addons": "^5.2.5",
     "@storybook/core": "5.3.0-alpha.41",
     "core-js": "^3.0.1",
     "global": "^4.3.2",
     "regenerator-runtime": "^0.13.3",
-    "ts-dedent": "^1.1.0"
+    "ts-dedent": "^1.1.0",
+    "webpack": "^4.33.0"
```

Comment:
why?

Reply:
I added webpack to be able to import the `Configuration` type
```
import { Configuration } from 'webpack';
export function webpack(config: Configuration) {
```
in the `framework-preset-svelte`. What would be the best alternative to this? @shilman

Reply:
Does webpack now ships its own types? There's also https://www.npmjs.com/package/@types/webpack

In app/angular @ndelangen added it to devDeps and peerDeps along with `@types/webpack-env` 🤔  (according to git blame)
https://github.com/storybookjs/storybook/blame/next/app/angular/package.json

In app/react `webpack` it is a direct dependency and also includes `@types/webpack` as a devDep

in app/vue it's the same as in app/react

*Back to this PR:* would it work with only `@types/webpack` as devDependency? 

Shipping new direct dependencies is sometimes the root cause of new issues. Do svelte apps install it's own webpack dependency? If yes, then we should ship it as a peerDependency. If not then something under the hood of svelte might already ship it as a direct dependency (kind of like angular)

@gaetanmaisse Can you assist on this as well?

Reply:
I think there is no need to add `webpack` as `dependency` neither as `devDependency`. What leads me to that is that: 
 - `Configuration` type is used in `server` part of `@storybook/svelte` which is not exported to the outside world via `types/main` properties of `package.json`, only `client` is.
 - This package does not use `webpack` directly, it just override/create a webpack config. 
 - This package has `@storybook/core` as a dependency, the latter uses `webpack` and have it as `dependencies`

However, just removing `webpack` from `package.json` leads to a [no-extraneous-dependencies](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md) eslint error in `server/framework-preset-svelte.ts`. But maybe our context is a good reason to disable this rule for this import?

We should maybe add `@types/webpack` as `devDependency` as we are using `Configuration` type in `svelte/src/server` codebase 🤷‍♂ even if everything looks good to TS compiler without adding it (surely because `@types/webpack` is required by another module of SB monorepo).

What's your mind about it?

Reply:
I'm fine with ignoring the `no-extraneous-dependencies` warning

Reply:
Rethinking about it, this issue will maybe be solved when `@storybook/core` will be migrated to TS.

If `@storybook/core` defines and exports a `WebpackConfiguration` type:
```ts
import { Configuration } from 'webpack';

export type WebpackConfiguration = Configuration
```
then using it in `@storybook/svelte` will not cause `no-extraneous-dependencies` as `@storybook/core` is a dependency. And everything should be good for TS if `@storybook/core` will have `webpack` and `@types/webpack` as dependencies.

Reply:
What would be the best approach for the meantime? Should we mark `config:any` until `@storybook/core` is migrated, or leave it as is until it is migrated?

Reply:
@aromanarguello You can add the comment to ignore `no-extraneous-dependencies` rule for this line, remove `webpack` from dependencies and 🚀